transform-plugins/widgets/XMLToJSON-transform.json

```diff
@@ -0,0 +1,24 @@
+{
+  "metadata": {
+    "spec-version": "1.0"
+  },
+  "configuration-groups": [
+    {
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Input Field",
+          "name": "field"
+        }
+      ]
+    }
+  ],
+  "outputs": [
+    {
+      "widget-type": "non-editable-schema-editor",
```

Comment:
so this plugin won't handle the case, where the input schema has 10 fields, of which we only want to convert one of the field from xml to json and keep the other 9 fields as is. we should allow that right ?